### ISSUE TITLE
String comments and function variables in Zed Python

### ISSUE THREAD

**Issue by maxjmohr** [catppuccin/zed#63]
Still many differences compared to the neovim version (Catppuccin Frappé).

Neovim:
![image](https://github.com/user-attachments/assets/55e6e2c1-9819-48ca-8f9d-4e0e9d4df941)

Zed:
![image](https://github.com/user-attachments/assets/b79d088f-5ce5-4a9b-a3b2-bb1894dc46f3)

Especially:
- String comments
- Function variables

**Comment by uncenter** [comment]
(aka doc comments and variable parameters)

**Comment by uncenter** [comment]
It should be mentioned that differences in the appearance of code syntax highlighting can also be due to *editor* differences, and not the theme. Specifically I believe some of these issues are due to differences in Zed's Tree-sitter *queries*, if you are familiar with those. Could you share the code snippet in this screenshot so I can reference it in testing?

**Comment by maxjmohr** [comment]
@uncenter Yeah I get what you are saying. I could probably also adjust the doc comments and variable parameters syntax highlighting somewhere in my config, but this is of course not ideal for other users.

```python
def regression(
    data: pd.DataFrame,
    regression: str = "bias_detected ~ C(scenario)",
    reg_type: str = "ols",
) -> None:
    "Regression to find out impact of scenarios on bias detections"
    # Regression
    if reg_type == "ols":
        model = smf.ols(regression, data=data)
    elif reg_type == "mixedlm":
        # Create column with bias + model
        data["biasmodel"] = data["bias"] + data["model"]

        model = smf.mixedlm(regression, data, groups=data["biasmodel"])
    else:
        raise ValueError("Invalid regression type")

    # Fit the model
    results = model.fit()

    # Print the formula bold
    print(
        "\n#----------------------------------------------------------------------------#"
    )
    print("####### Regression formula: #######")
    print(f"\033[1m{regression}\033[0m")
    print("\n####### Regression results: #######")
    print(results.summary())
```

**Comment by uncenter** [comment]
The second issue you mention with function variables is a query issue in Zed and not fixable by us.

Neovim queries:

https://github.com/nvim-treesitter/nvim-treesitter/blob/e947d35bbf919ea88a24c21cb8afa4621e3cf547/queries/python/highlights.scm#L114-L167

Zed doesn't have these, trying to get the Zed team to start adopting query captures like `variable.parameter` in https://github.com/zed-industries/zed/issues/22193 though.
